api/controllers/file_processor/file-base64-upload.js

```diff
@@ -0,0 +1,144 @@
+/**
+ * file_processor/file-base64-upload.js
+ *
+ *
+ * @api {post} /file/upload/base64/:objID/:fieldID Upload
+ * @apiParam {string: base64} file
+ * @apiParam {uuid} photoID
+ * @apiParam {uuid} objID
+ * @apiParam {uuid} fieldID
+ * @apiParam {string} uploadedBy
+ * @apiParam {number} size
+ * @apiParam {string} type
+ * @apiParam {string} fileName
+ * @apiUse successRes
+ * @apiSuccess (200) {Object} data
+ * @apiSuccess (200) {string} data.uuid
+ */
+
+const async = require("async");
+const { error } = require("console");
+
+var inputParams = {
+   file: { string: true, required: true },
+   photoID: { string: true, required: true },
```

Comment:
Change to `fileID` to be consistent? Not necessarily a photo.

Reply:
Also should this be optional? Normally we don't expect the client to assign the `uuid`. Making this optional would make the route more flexible for other use cases.